### DISCUSSION TITLE
Changes in locales / messages.json

### DISCUSSION THREAD

**Discussion by czarnetz** [dani-garcia/vaultwarden#3621] [General]
Dear,

if i change some translatations in e.G. "locales/de/messages.json" what i have to do, that the changes will be affected?
It looks like, that the content will be cached :-/

Thx
Thomas

**Comment by BlackDex** [comment]
Vaultwarden does not have translations.

If you mean the web-vault translations you should go to Bitwarden instead.

I'm not sure on how to contribute translations actually, this is a link to there localization page.
https://bitwarden.com/help/localization/#dont-see-your-language
